samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_app_utils.c

```diff
@@ -15,18 +15,18 @@ LOG_MODULE_REGISTER(app_lwm2m_utils, CONFIG_APP_LOG_LEVEL);
 
 void set_ipso_obj_timestamp(int ipso_obj_id, unsigned int obj_inst_id)
 {
-	int32_t timestamp;
+	time_t timestamp;
 	char path[MAX_LWM2M_PATH_LEN];
 
-	int ret = lwm2m_engine_get_s32(LWM2M_PATH(LWM2M_OBJECT_DEVICE_ID, 0,
+	int ret = lwm2m_engine_get_time(LWM2M_PATH(LWM2M_OBJECT_DEVICE_ID, 0,
 					CURRENT_TIME_RID), &timestamp);
 	if (ret) {
 		LOG_ERR("Unable to retrieve timestamp");
 	}
 
 	snprintk(path, MAX_LWM2M_PATH_LEN, "%d/%u/%d", ipso_obj_id, obj_inst_id, TIMESTAMP_RID);
 
-	ret = lwm2m_engine_set_s32(path, timestamp);
+	ret = lwm2m_engine_set_time(path, timestamp);
```

Comment:
```suggestion
	ret = lwm2m_engine_set_time(path, time(NULL));
```

Can we just use `time(NULL)` instead of trying to copy the timestamp from device object. We either have a clock, or we don't.

Reply:
Done